crypto/pem/pem_lib.c

```diff
@@ -505,7 +505,7 @@ int PEM_get_EVP_CIPHER_INFO(char *header, EVP_CIPHER_INFO *cipher)
         return 0;
     header += strspn(header, " \t");
 
-    /* We expect "ENCRYPTED" followed by optional white-space + line break */
```

Comment:
I believe white-space and whitespace are both valid terminology. "white space" is not, so those are good to change, but I think keeping "white-space" is fine.